{
    "type": "bundle",
    "id": "bundle--324a65bd-57bb-4e17-8716-8a60b5780169",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--ea3b661b-511a-4052-9594-40e797005918",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2022-09-17T00:06:08.572448Z",
            "modified": "2022-09-17T00:06:08.572448Z",
            "name": "CVE-2022-36001",
            "description": "TensorFlow is an open source platform for machine learning. When `DrawBoundingBoxes` receives an input `boxes` that is not of dtype `float`, it gives a `CHECK` fail that can trigger a denial of service attack. We have patched the issue in GitHub commit da0d65cdc1270038e72157ba35bf74b85d9bda11. The fix will be included in TensorFlow 2.10.0. We will also cherrypick this commit on TensorFlow 2.9.1, TensorFlow 2.8.1, and TensorFlow 2.7.2, as these are also affected and still in supported range. There are no known workarounds for this issue.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-36001"
                }
            ]
        }
    ]
}